config/send.js

```diff
@@ -0,0 +1,9 @@
+import admin from 'firebase-admin';
+import serviceAccount from '../evently-firebase.json';
+
+// const serviceAccount = require('path/to/serviceAccountKey.json');
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: 'https://evently-264812.firebaseio.com'
```

Comment:
Can we have this in the environment variable?

Reply:
wokred on this thanks